websites/R/RTLplay/presence.ts

```diff
@@ -0,0 +1,555 @@
+		case ["ma-liste"].includes(pathname.split("/")[2]): {
+			presenceData.details = (await strings).browsing;
+			presenceData.state = (await strings).viewAPage;
+			if (!privacy) {
+				presenceData.state = (await strings).viewList;
+				if (buttons) {
+					presenceData.buttons = [
+						{
+							label: (await strings).buttonViewPage, // Need to be a general string
+							url: href, // We are not redirecting directly to the raw video stream, it's only the media page
+						},
+					];
+				}
+			}
+			break;
+		}
+
+		/* CATEGORY PAGE / COLLECTION PAGE (Page de catégorie)
+
+		(https://www.rtlplay.be/rtlplay/collection/c2dBY3Rpb24) */
+		case ["collection", "series", "films", "divertissement"].includes(
+			pathname.split("/")[2]
+		): {
+			if (privacy) presenceData.details = (await strings).viewAPage;
+			else {
+				const data = JSON.parse(
+					document.querySelectorAll("script[type='application/ld+json']")[0]
+						.textContent
+				);
+
+				presenceData.details = (await strings).viewCategory;
+				presenceData.state =
+					pathname.split("/")[2] !== "collection"
+						? pathname.split("/")[2][0].toUpperCase() +
+						  pathname.split("/")[2].substring(1) // to Upper Case the first letter
+						: data[0]["@type"] === "CollectionPage"
+						? data[0].name
+						: null;
+
+				presenceData.smallImageKey = Assets.Viewing;
+				presenceData.smallImageText = (await strings).viewCategory;
+
+				if (time) presenceData.startTimestamp = browsingTimestamp;
+
+				if (buttons) {
+					presenceData.buttons = [
+						{
+							label: (await strings).buttonViewPage, // Need to be a general string
+							url: href, // We are not redirecting directly to the raw video stream, it's only the media page
+						},
+					];
+				}
+			}
+			break;
+		}
+
+		/* DIRECT PAGE (Page des chaines en direct)
+
+		(https://www.rtlplay.be/rtlplay/direct/tvi)*/
+		case (hostname === "www.rtlplay.be" &&
+			pathname.split("/")[2] === "direct") ||
+			(hostname === "www.radiocontact.be" &&
+				pathname.split("/")[1] === "player"): {
+			switch (true) {
+				case hostname === "www.rtlplay.be": {
+					if (exist("i.playerui__icon--name-play")) {
+						// State paused
+						presenceData.smallImageKey = Assets.Pause;
+						presenceData.smallImageText = (await strings).pause;
+					} else if (exist("div.playerui__liveStat--deferred")) {
+						// State deferred
+						presenceData.smallImageKey = Assets.Deferred;
+						presenceData.smallImageText = "En Différé"; // Need to be a general string
+					} else {
+						// State live
+						presenceData.smallImageKey = Assets.LiveAnimated;
+						presenceData.smallImageText = (await strings).live;
+					}
+
+					if (privacy) {
+						presenceData.details = (await strings).watchingLive;
+						presenceData.largeImageKey = Assets.Logo;
+					} else {
+						if (
+							!useChannelName &&
+							document
+								.querySelectorAll("li[aria-current='true'] > a > div > h2")[0]
```

Comment:
If you're getting the first element user querySelector() instead of querySelectorAll()
```suggestion
								.querySelector("li[aria-current='true'] > a > div > h2")
```